docs/content/1.get-started/1.installation.md

```diff
@@ -63,6 +63,10 @@ div {
 
 That's it! Nuxt Fonts will detect this and you should immediately see the web font loaded in your browser. [Read more about how it works](/advanced#how-it-works).
 
+::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
```

Comment:
This icon/color combo is pulled from the Nuxt UI docs; not sure if the icon set used here is available in the docs for `@nuxt/fonts`